colors.scss

```diff
@@ -476,53 +476,53 @@ $blk-235: #060606;
 $blk-240: #000000;
 
 // UI Colors Light Theme
-$ui-blue: $h-bb-060;
-$ui-blue-hover: $h-bb-050;
-$ui-blue-press: $h-bb-070;
-$ui-green: $h-gg-060;
-$ui-green-hover: $h-gg-050;
-$ui-green-press: $h-gg-070;
-$ui-yellow: $h-yy-060;
-$ui-yellow-hover: $h-yy-050;
-$ui-yellow-press: $h-yy-070;
-$ui-red: $h-rr-060;
-$ui-red-hover: $h-rr-050;
-$ui-red-press: $h-rr-070;
+$ui-blue-1: $h-bb-060;
+$ui-blue-2: $h-bb-050;
+$ui-blue-3: $h-bb-070;
+$ui-green-1: $h-gg-060;
+$ui-green-2: $h-gg-050;
+$ui-green-3: $h-gg-070;
+$ui-yellow-1: $h-yy-060;
+$ui-yellow-2: $h-yy-050;
+$ui-yellow-3: $h-yy-070;
+$ui-red-1: $h-rr-060;
+$ui-red-2: $h-rr-050;
+$ui-red-3: $h-rr-070;
 $ui-background: $blk-005;
-$ui-foreground: $blk-000;
-$ui-foreground-hover: $blk-010;
-$ui-foreground-press: $blk-020;
+$ui-foreground-1: $blk-000;
+$ui-foreground-2: $blk-010;
+$ui-foreground-3: $blk-020;
 $ui-text-1: $blk-220;
-$ui-text-2:$blk-170;
+$ui-text-2: $blk-170;
 $ui-text-3: $blk-140;
 $ui-border-1: $blk-050;
 $ui-border-2: $blk-030;
 $ui-border-3: $blk-020;
-$ui-border-hover: $blk-090;
-$ui-border-press: $blk-130;
+$ui-border-4: $blk-090;
```

Comment:
In case it's not already been done yet, might be good to create an issue to add these Dark Theme hex values to the vars.

Reply:
Ugh...see line 505 and below. 🤦🏾‍♂️